src/hansken-js.js

```diff
@@ -2,10 +2,12 @@ import { ProjectContext } from './modules/projectContext.js';
 import { SinglefileContext } from './modules/singefileContext.js';
 import { Scheduler } from './modules/scheduler.js';
 import { SessionManager } from './modules/sessionManager.js';
+import { TraceModelContext } from './modules/traceModelContext.js';
 
 class HanskenClient {
 
     #scheduler;
+    #traceModelContexts = {}; // {projectId, traceModelContext}
```

Comment:
what does to comment add?

Reply:
What the key and value are of the object